### DISCUSSION TITLE
ERROR  react-native-compat: Application module is not available

### DISCUSSION THREAD

**Discussion by musaddiqaskira** [WalletConnect/walletconnect-monorepo#5110] [Developer Support] [answered]
install the module and imported it on the expo app but still getting this error:  ERROR  react-native-compat: Application module is not available. Can someone help please

**Comment by glitch-txs** [comment]
cc @ignaciosantise

**Comment by ignaciosantise** [comment, marked as answer]
Hey! Did you also install "expo-application"?

**Reply by musaddiqaskira** [reply]
Hi, thanks a lot. Just installed expo-application and everything works!!!.
